src/infrastructure/NodeService.js

```diff
@@ -168,87 +168,70 @@ class NodeService {
 
 	static getNodeInfo = async publicKey => {
 		try {
-			const node = await http.statisticServiceRestClient().getNode(publicKey);
+			const node = await NodeWatchService.getNodeByMainPublicKey(publicKey);
+
 			const formattedNode = this.formatNodeInfo(node);
 
-			if (formattedNode?.apiStatus) {
-				const {
-					finalization,
-					chainHeight,
-					lastStatusCheck,
-					nodeStatus,
-					isAvailable,
-					isHttpsEnabled,
-					restVersion
-				} = formattedNode.apiStatus;
+			const {
+				finalizedEpoch,
+				finalizedHash,
+				finalizedHeight,
+				finalizedPoint,
+				height,
+				isHealthy,
+				isHttpsEnabled,
+				restVersion
+			} = formattedNode;
+
+			// Chain info
+			formattedNode.chainInfo = {
+				height: height,
+				finalizedHeight: finalizedHeight,
+				finalizationEpoch: finalizedEpoch,
+				finalizationPoint: finalizedPoint,
+				finalizedHash: finalizedHash
+			};
+
+			formattedNode.apiStatus = {};
 
-				// Api status
+			// Only API nodes have database status
+			if ([2, 3, 6, 7].includes(formattedNode.rolesRaw)) {
 				formattedNode.apiStatus = {
-					connectionStatus: isAvailable,
+					connectionStatus: isHealthy || Constants.Message.UNAVAILABLE,
 					isHttpsEnabled,
-					restVersion,
-					lastStatusCheck: moment
-						.utc(lastStatusCheck)
-						.format('YYYY-MM-DD HH:mm:ss')
+					restVersion
 				};
-
-				// Only API nodes have database status
-				if ([2, 3, 6, 7].includes(node.roles)) {
-					formattedNode.apiStatus = {
-						...formattedNode.apiStatus,
-						apiNodeStatus:
-							'up' === nodeStatus?.apiNode || Constants.Message.UNAVAILABLE,
-						databaseStatus: 'up' === nodeStatus?.db || Constants.Message.UNAVAILABLE
-					};
-				} else {
+			} else {
+				if (null != restVersion) {
 					formattedNode.apiStatus = {
-						...formattedNode.apiStatus,
-						lightNodeStatus: isAvailable || Constants.Message.UNAVAILABLE
+						lightNodeStatus: null != restVersion,
```

Comment:
I am assuming ``null != restVersion`` is always true here?

Reply:
would it make sense to do this ``lightNodeStatus: null != restVersion`` => ``lightNodeStatus: true`` ?